dev-libs/xxhash/xxhash-0.6.4.recipe

```diff
@@ -0,0 +1,67 @@
+SUMMARY="A fast non-cryptographic hash algorithm"
+DESCRIPTION="xxHash is an extremely fast non-cryptographic hash algorithm, working at speeds close to RAM limits."
+HOMEPAGE="http://cyan4973.github.io/xxHash/"
+COPYRIGHT="2012-2016 Yann Collet"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://github.com/Cyan4973/xxHash/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="4570ccd111df6b6386502791397906bf69b7371eb209af7d41debc2f074cdb22"
+SOURCE_DIR="xxHash-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="0"
+
+PROVIDES="
+	libxxhash$secondaryArchSuffix = $portVersion
+	cmd:xxhsum$secondaryArchSuffix = $portVersion
+	lib:libxxhash$secondaryArchSuffix = $libVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libxxhash${secondaryArchSuffix}_devel = $portVersion
+	devel:libxxhash$secondaryArchSuffix = $libVersion
+	"
+REQUIRES_devel="
+	libxxhash$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+defineDebugInfoPackage libxxhash$secondaryArchSuffix \
+	"$binDir"/xxhsum \
+	"$libDir/libxxhash.so.$libVersion"
+
+BUILD()
+{
+	make $jobArgs
+}
+
+INSTALL()
+{
+	install -d -m 755 "$binDir" "$libDir" "$includeDir"
+
+	# header
+	install -t "$includeDir" -m 644 xxhash.h
+
+	# lib
+	install -t "$libDir" -m 755 libxxhash.so.$libVersion
+	ln -s libxxhash.so.$libVersion "$libDir/libxxhash.so"
+	install -t "$libDir" -m 644 libxxhash.a
+
+	# bins
+	install -t "$binDir" -m 755 xxhsum
```

Comment:
You could use `make install`, which would additionally package the manpages for us

```diff
-	install -d -m 755 "$binDir" "$libDir" "$includeDir"
-
-	# header
-	install -t "$includeDir" -m 644 xxhash.h
-
-	# lib
-	install -t "$libDir" -m 755 libxxhash.so.$libVersion
-	ln -s libxxhash.so.$libVersion "$libDir/libxxhash.so"
-	install -t "$libDir" -m 644 libxxhash.a
-
-	# bins
-	install -t "$binDir" -m 755 xxhsum
+	make BINDIR="$binDir" INCLUDEDIR="$includeDir" LIBDIR="$libDir" \
+		MANDIR="$manDir/man1" install

Reply:
Good suggestion, will test it.

Reply:
We need to patch the supplied Makefile, as it specifically disables the install target for unsupported platforms. By adding "Haiku" to the whitelisted supported platforms, make install becomes available and works as expected.

I've included a further patch in the recipe - please check.